src/Adapter/Cart/CartPresenter.php

```diff
@@ -283,7 +286,20 @@ public function present($cart)
             'id_address_delivery' => $cart->id_address_delivery,
             'id_address_invoice' => $cart->id_address_invoice,
             'vouchers' => $this->getTemplateVarVouchers($cart),
-        ];
+            'minimalPurchaseRequired' =>
+                ($this->priceFormatter->convertAmount($productsTotalExcludingTax) < $minimalPurchase) ?
+                sprintf(
+                    $this->translator->trans(
+                        'A minimum shopping cart total of %s (tax excl.) is required to validate your order.
+                        Current cart total is %s (tax excl.).',
+                        [],
+                        'Cart'
+                    ),
+                    $this->priceFormatter->convertAndFormat($minimalPurchase),
+                    $this->priceFormatter->convertAndFormat($productsTotalExcludingTax)
```

Comment:
We know it can be confusing, but we can't have a limitation with tax included because sometimes you could have several tax rates for the same shop selling in different countries. So it's impossible to know for which tax rate the minimal price is set, so we have to set it without taxes.